styles.css

```diff
@@ -271,5 +271,119 @@ nav {
     padding: 20px;
 }
 
+<<<<<<< HEAD
+/*************************************************************/
+
+/***************** Back to the Future II *********************/
+.body__p2 {
+
+    background-image: url(photos/NBP-grit-13\ copy.jpg);
```

Comment:
https://www.filmstories.co.uk/wp-content/uploads/2019/10/back-to-the-future.jpg

Reply:
this is my background texture, not eh back to the future image

Reply:
Sorry I typed that fast =/